extensions/sdk-assistant/README.md

```diff
@@ -0,0 +1,60 @@
+# Posit Connect SDK Assistant
+
+![SDK Assistant](sdk-assistant.gif)
+
+This is a tool to help developers understand how to use the [Posit Connect SDK python package](https://github.com/posit-dev/posit-sdk-py).
+
+Currently, it does not execute any code.
+
+## Usage
+
+To view the live application, visit [this link](https://connect.posit.it/sdk-assistant/).
```

Comment:
This isn't available publicly is it? Is there a place we could host it publicly?

Reply:
Or maybe we remove the link to it if not? Either is ok

Reply:
I'll remove it as it's at least linked in the PR and we do not have plans to allow for `connect.posit.it` apps to be public.